src/encoded/static/components/filegallery.js

```diff
@@ -1187,6 +1237,14 @@ FilterControls.defaultProps = {
     currentBrowser: '',
 };
 
+FilterControls.propTypes = {
+    /** Object within which the attachment is to be displayed */
+    context: PropTypes.object.isRequired,
+}
```

Comment:
You should just add `context` to the existing `FilterControls.propTypes` object instead of overwriting the existing one with this one.

Reply:
Done.